src/tests/suites/KubectlClientProvider-Test.ts

```diff
@@ -0,0 +1,70 @@
+import { expect } from 'chai';
+import { after, describe, it } from 'mocha';
+import * as process from 'process';
+import { CommandRunner } from '../../clients/CommandRunner';
+import { IClientProvider } from '../../clients/Providers/IClientProvider';
+import { loggerStub } from '../CommonTestObjects';
+import { BinariesVersionClient } from '../../clients/BinariesVersionClient';
+import { KubectlClientProvider } from '../../clients/Providers/KubectlClientProvider';
+import { AccountContextManager } from '../../models/context/AccountContextManager';
+
+describe('KubectlClientProviderTest', () => {
+    const originalPlatform = Object.getOwnPropertyDescriptor(process, 'platform');
+    it('should always return linux/kubectl for linux', async () => {
+        Object.defineProperty(process, 'platform', {
+            value: 'linux'
+        });
+        const expectedName = 'linux/kubectl';
```

Comment:
@Vidya2606 I think you missed calling the actual method here , can you check please ? ex: 
```
 kubectlClientProvide.getExecutableFilePath(); 
```